recipe/meta.yaml

```diff
@@ -48,4 +47,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - CJ-Wright
+    - st3107
```

Comment:
It was too fast :)
This block of extra -> recipe-maintainers should be completely removed from the recipe in this organization.